packages/typescript/test/member-chain.test.tsx

```diff
@@ -119,6 +130,153 @@ it("works without function call expressions", () => {
       { printWidth: 5 },
     ),
   ).toBe(d`
-    z.string.min
+    z.string
+    .min
```

Comment:
If a linebreak is introduced here (which is fine), the next line should be indented.